clients/android/NewsBlur/src/com/newsblur/activity/Reading.java

```diff
@@ -771,12 +771,34 @@ private boolean isVolumeKeyNavigationEvent(int keyCode) {
     private void processVolumeKeyNavigationEvent(int keyCode) {
         if ((keyCode == KeyEvent.KEYCODE_VOLUME_DOWN && volumeKeyNavigation == VolumeKeyNavigation.DOWN_NEXT) ||
             (keyCode == KeyEvent.KEYCODE_VOLUME_UP && volumeKeyNavigation == VolumeKeyNavigation.UP_NEXT)) {
-            overlayRight(overlayRight);
+            if (overlayNavigationSupported()) {
+                overlayRight(overlayRight);
+            } else {
+                int nextPosition = pager.getCurrentItem() + 1;
+                if (nextPosition < readingAdapter.getCount()) {
+                    pager.setCurrentItem(nextPosition);
```

Comment:
We might need to do some additional checking to ensure that the pager isn't null, and that setCurrentItem() could still throw exceptions despite the check - the async nature of cursor updates makes it possible for the number of available pages to change out from under you in the time it takes to run just one line of code.